tests/integration/components/partial-move-input-test.js

```diff
@@ -0,0 +1,35 @@
+import { test, moduleForComponent } from 'ember-qunit';
```

Comment:
@steveyken , we can add mock data but checking for line items won't be possible, as you can see to even check the component I'd to render that component in beforeEach block.
- If we want to check for line-items then we would have to render other buttons message-box and may be entire page.
- So in integration testing, I think we can only write test cases which are related to component but we can't preform any other operations for which we need to render other details of page
- As for moving quantity for packages in different locations, I've already covered that in partial-move-test.js file
- Also we're dealing with singleton packages only, so it is not possible to move partial quantity so there would be always **one line item only**, so that's why scenarios for test-cases are few.

Let me know if any scenarios are still missing or there are other concerns :)
Thanks